confidential-identity/Cargo.toml

```diff
@@ -29,7 +30,7 @@ rand_core = { version = "0.5", default-features = false, features = ["alloc"] }
 rand = { version = "0.7", default-features = false, features = ["alloc"] }
 getrandom = { version = "0.1.14", default-features = false, optional = true }
 
-curve25519-dalek = { git = "https://github.com/PolymathNetwork/curve25519-dalek.git", branch = "v2-packed-simd", default-features = false, features = ["nightly", "alloc"] }
+curve25519-dalek = { git = "https://github.com/PolymathNetwork/curve25519-dalek.git", branch = "v2-packed-simd", default-features = false, features = ["nightly", "alloc", "u64_backend"] }
```

Comment:
From a general perspective of the library is better to avoid any fixed back-end, like `u64_backend` because the user of the library should be free to decide it, i.e. someone could try to use this in a 32 bit machine.
In this case it is not really an issue, even with `avx2_backend` which includes `u64_backend`.

My suggestion is to remove it from here, and force the user to select one backend.